src/TodoFilter.js

```diff
@@ -0,0 +1,54 @@
+import React, {useState} from "react";
+import TodoCount from "./TodoCount";
+import TodoList from "./TodoList";
+import {FILTER_TYPE} from "./constants";
+
+const TodoFilter = ({onFilter}) => {
+
+  const [selectAll, setSelectAll] = useState(true);
+  const [selectActive, setSelectActive] = useState(false);
+  const [selectCompleted, setSelectCompleted] = useState(false);
```

Comment:
```suggestion
  const [selected, setSelected]  = useState(FILTER_TYPE.ALL)
```

이런 식으로 로직을 변경하면 중복되는 코드 수를 줄일 수 있을 것 같네요!